cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -0,0 +1,56 @@
+import functions_framework
+import pathlib
+
+from cloudevents.http import CloudEvent
+from google.cloud import firestore
+from typing import Optional
+
+STUDY_AREAS = "study_areas"
```

Comment:
Nit: Would be clearer to call this STUDY_AREAS_ID or STUDY_AREAS_COLLECTION_ID.

Reply:
Done